install/README.md

```diff
@@ -87,6 +87,7 @@ available observability plugins, to minimize the disk space used for install.
 - [Knative Install on Minikube](Knative-with-Minikube.md)
 - [Knative Install on Minishift](Knative-with-Minishift.md)
 - [Knative Install on OpenShift](Knative-with-OpenShift.md)
+- [Knative Install via Operator](https://github.com/openshift-cloud-functions/knative-operators)
```

Comment:
Is this correct, or should we link to https://github.com/openshift-cloud-functions/Documentation/blob/master/knative-OCP-4x.md ?

Reply:
@abrennan89 yes that seems much better, I've just changed it.